### ISSUE TITLE
Test failures due to Credential Manifest issues/

### ISSUE THREAD

**Issue by rolsonquadras** [trustbloc/sandbox#1500]
Some tests using CHAPI share fails with below errors.

`Error: agent wasm: code: 12016, message: failed to read credential manifest: invalid credential manifest: display description for output descriptor at index 1 is invalid: display mapping object must contain either a paths or a text property`

**Comment by rolsonquadras** [comment]
https://github.com/trustbloc/k8s/blob/main/wallet-web/kustomize/wallet-web/overlays/common/static/config/credentialDisplayData.json

**Comment by HeidiHan0000** [comment]
There was a fix I made earlier regarding VCs which were missing the name property, and as a result, wallet would display a credential card with a blank name and that was a problem. The solution was to look for the [default output descriptors ](https://github.com/trustbloc/wallet/blob/main/cmd/wallet-web/src/config/credential-output-descriptors.json) in wallet and replace the missing properties with the default ones. What I did not know or consider was that not all credentials in the sandbox test cases have a default credential manifest in wallet. For those, the credential manifests will still get overridden, then in `resolveManifest`  and `validate` will fail. I will make the fix in wallet and hopefully this will resolve the issue.
@rolsonquadras @fqutishat
